### ISSUE TITLE
LLVM Instrumentation

### ISSUE THREAD

**Issue by walsht** [urbanairship/ios-library#151]
Got an invalid binary when pushing to Apple because your framework has "Gather coverage date" checked on your test target.  Please update. 
https://developer.apple.com/library/content/qa/qa1964/_index.html#//apple_ref/doc/uid/DTS40017675
https://developer.apple.com/library/content/qa/qa1964/_index.html#//apple_ref/doc/uid/DTS40017675-CH1-INSPECT

**Comment by rlepinski** [comment]
@walsht How are you installing the SDK? Cocoapods, manual, or carthage?

**Comment by walsht** [comment]
carthage

**Comment by walsht** [comment]
https://github.com/urbanairship/ios-library/pull/152

**Comment by rlepinski** [comment]
We have the change internally. Working on doing a release right now. Looks like this is a carthage issue https://github.com/Carthage/Carthage/issues/2056.

**Comment by marc-scig** [comment]
@walsht did you build this with the Xcode 9 beta?

**Comment by marc-scig** [comment]
(Or Xcode 9 in general)

**Comment by walsht** [comment]
XCode GM

**Comment by walsht** [comment]
Not technically a Carthage issue, just a new thang Apple instilled on us.

**Comment by marc-scig** [comment]
Yeah, found this in the release notes (posting here for posterity)

```
Schemes with Code Coverage enabled (in the Test action) will now always build with Code
Coverage, including for the Run and Launch actions. This reduces the need to build multiple
versions of the application. Users who require code coverage to be off for their non-test actions
can explicitly use a separate scheme with coverage disabled. (31848014)
```

Sorry for the trouble, we'll get this updated as soon as we can.

**Comment by walsht** [comment]
No worries updated the Carthage framework manually and ran `carthage build ios-library`
Processing now with Apple.

**Comment by rlepinski** [comment]
Fixed in 8.6.0
